src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/OrderSummary/OrderSummaryController.ts

```diff
@@ -67,6 +67,9 @@ module Orckestra.Composer {
                     cancelEditOrder() {
```

Comment:
I think need to add before 

this.Mode.Loading = true;
then on fin(() =>  this.Mode.Loading = false)
also need to define new error and output some messegae;